packages/aws-rfdk/lib/deadline/lib/spot-event-plugin-fleet.ts

```diff
@@ -418,6 +437,16 @@ export class SpotEventPluginFleet extends Construct implements ISpotEventPluginF
    */
   public readonly blockDevices?: BlockDevice[];
 
+  /**
+   * The Launch Template for this Spot Fleet. This launch template does not specify an instance type or subnet.
+   */
+  public readonly launchTemplate: LaunchTemplate;
+
+  /**
+   * @internal
+   */
+  public readonly _launchTemplateConfigs: any[];
```

Comment:
Can we type this?

```suggestion
  public readonly _launchTemplateConfigs: LaunchTemplateConfig[];
```

Reply:
Why does this start with an underscore?

Reply:
Hence the `@internal` tag above. Looks like [JSII might filter out such fields](https://aws.github.io/jsii/user-guides/lib-author/typescript-restrictions/):

>   ... they do not apply to any internal declarations such as `private` members and declarations annotated with the `/** @internal */` tag.

Reply:
Experimented with our documentation generation and it omits this b/c it is marked `@internal`.  I'm okay with keeping the `public` access modifier along with `@internal` and the signaling you've added to communicate that this is an internal API.

Resolving.